specification/mariadb/resource-manager/readme.cli.md

```diff
@@ -0,0 +1,64 @@
+## CLI
+
+These settings apply only when `--cli` is specified on the command line.
+
+``` yaml $(cli)
```

Comment:
Shall we remove '$(cli)' here?   The current clicommon will ignore this config if we have $(cli) here. So it may just be:
```
``` yaml
```

And the description in Line3 need to be modified accordingly.

Reply:
I can remove `$(cli)`. 
Can you show me what description can be showed , if not this one.

Reply:
Thanks. I think the description can be:
```
These settings don't need to apply `--cli` on the command line.
```